pkgs/development/interpreters/gnudatalanguage/default.nix

```diff
@@ -0,0 +1,167 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+# Native build inputs for the compiler
+, cmake
+, wrapGAppsHook
+# Build dependencies
+, readline
+, ncurses # alt: curses
+, zlib
+, gsl
+, openmp
+# GraphicsMagick are preferred over ImageMagick
+# https://github.com/gnudatalanguage/gdl/blob/master/CMakeLists.txt#L600-L601
+, graphicsmagick # alt: imagemagick
```

Comment:
This info is not much useful. The presence of graphicsmagick already suggests by itself it is the preferred choice.